src/api/routes/articlesRouter.js

```diff
@@ -19,4 +27,12 @@ articlesRouter.put('/:slug', isAuth.verifyToken, check.articleOwner, validate.up
 
 articlesRouter.delete('/:slug', isAuth.verifyToken, check.articleOwner, deleteArticle);
 
+// Comments routes
+
+articlesRouter.post('/:slug/comments', isAuth.verifyToken, validateBody('comment'), articleExists, checkComment, createComment);
+articlesRouter.post('/:slug/comments/:commentId', isAuth.verifyToken, validateBody('comment'), articleExists, checkComment, commentAcomment);
+articlesRouter.patch('/comments/:commentId', isAuth.verifyToken, validateBody('comment'), checkParameter, editComment);
+articlesRouter.delete('/comments/:commentId', isAuth.verifyToken, checkParameter, deleteComment);
```

Comment:
/:slug/comments/:commentId

Reply:
I don't think this is necessary since we have a comments table.
This would be helpful if we were deleting in the articles table.